wiki/Game_mode/osu!/id.md

```diff
@@ -0,0 +1,285 @@
+
+## Play style
+
+*Mohon lihat laman [Gaya permainan](/wiki/Play_Styles) untuk informasi lebih lenjut seputar gaya-gaya permainan yang umum digunakan di osu!.*
+
+## Kontrol
+
+Secara *default*, input-input yang terdaftar pada osu! adalah sebagai berikut:
+
+| Mouse | Keyboard | Tablet/Touchscreen |
+| :-- | :-- | :-- |
+| Klik kiri(M1)/Klik kanan(M2) | `Z`(K1)/`X`(K2) | Sentuhan pada layar(M1) |
+
+Pada umumnya, osu! dapat menerima input dalam bentuk apapun dari peranti apapun selama kamu memberikan input yang bersangkutan pada waktu dan lokasi yang tepat.
+
+Apabila kamu bermain menggunakan mod [Relax](/wiki/Game_modifier/Relax), kamu hanya perlu menggerakkan kursormu tanpa harus memberikan input apapun ke dalam osu!. Sebaliknya, apabila kamu bermain dengan menggunakan mod [Auto Pilot](/wiki/Game_modifier/Autopilot), kamu hanya perlu memberikan input ke dalam osu! tanpa perlu menggerakkan kursormu. Kecepatan spinner pada mod AutoPilot memiliki kecepatan yang sama dengan mod [Spun Out](/wiki/Game_modifier/Spun_Out).
+
+## Penilaian
+
+*Mohon lihat laman [Penilaian skor pada mode osu!](/wiki/Score#osu) untuk informasi lebih lanjut seputar penilaian skor yang dipakai oleh osu!.*
+
+Berikut ini adalah penjelasan mengenai penilaian skor yang digunakan dipakai oleh osu! untuk menentukan skor kamu.
+
+### Peringkat
+
+| Peringkat | Kondisi |
+| :-: | :-- |
+| SS | Akurasi 100% |
+| S | Persentase perolehan 300 di atas 90%, persentase perolehan 50 di bawah 1% dan tidak ada miss |
+| A | Persentase perolehan 300 di atas 80% dan tidak ada miss **ATAU** perolehan 300 diatas 90% |
+| B | Persentase perolehan 300 di atas 70% dan tidak ada miss **ATAU** perolehan 300 diatas 80% |
+| C | Persentase perolehan 300 di atas 60% |
+| D | Lainnya |
+
+Peringkat S atau SS yang berwarna perak dapat diperoleh dengan memasang mod *[Hidden](/wiki/Game_modifier/Hidden)* dan/atau mod *[Flashlight](/wiki/Game_modifier/Flashlight)*.
+
+### Akurasi
+
+Pada mode permainan osu!, akurasi dihitung dengan menjumlahkan penilaian yang diperoleh pada setiap nilai hit dan membaginya dengan jumlah maksimal hit.
+
+Dengan kata lain: `Akurasi = Jumlah poin dari hit yang berhasil/(Jumlah maksimal hit * 300 poin)`
+
+| Istilah | Rumus |
+| :-: | :-- |
+| **Total poin hit** | `(jumlah perolehan poin yang bernilai 50 * 50) + (jumlah perolehan poin yang bernilai 100 * 100) + (jumlah perolehan poin yang bernilai 300 * 300)` |
+| **Total hit** | `Miss + jumlah perolehan poin yang bernilai 50 + jumlah perolehan poin yang bernilai 100 + jumlah perolehan poin yang bernilai 300` |
+
```

Comment:
missing `s` and `p` https://osu.ppy.sh/wiki/en/Game_mode/osu%21
![image](https://user-images.githubusercontent.com/39128490/120282570-70754e00-c2e4-11eb-9fdd-8cb9acd487b6.png)

Reply:
in Bahasa plurals aren't being expressed with an s at the end of the corresponding words; stuffs such as 300s and 100s shouldn't be used